crates/languages/src/typescript.rs

```diff
@@ -297,19 +305,13 @@ impl LspAdapter for EsLintLspAdapter {
 
     async fn fetch_latest_server_version(
         &self,
-        delegate: &dyn LspAdapterDelegate,
+        _delegate: &dyn LspAdapterDelegate,
     ) -> Result<Box<dyn 'static + Send + Any>> {
-        // We're using this hardcoded release tag, because ESLint's API changed with
-        // >= 2.3 and we haven't upgraded yet.
-        let release = github_release_with_tag(
-            "microsoft/vscode-eslint",
-            "release/2.2.20-Insider",
-            delegate.http_client(),
-        )
-        .await?;
+        let url = build_tarball_url("microsoft/vscode-eslint", Self::CURRENT_VERSION)?;
```

Comment:
Is Zed running VS Code extensions inside of it? And then, does Zed have a JavaScript engine inside to run this? _Fascinating_ 😃

Reply:
It's a lot more boring than that. `vscode-eslint` is a wrapper around ESLint, providing an implementation of LSP. What we do is to simply `node vscode-eslint/server/out/server.js` in the right directory, etc.

Reply:
I see :) thanks for the explanation